package exception_15.checkedexception.classe;

import java.io.FileNotFoundException;

import javax.security.auth.login.LoginException;

import exception_15.customexception.LoginInvalidoException;

public class Pessoa {
    public void salvar() throws LoginInvalidoException, FileNotFoundException{
        
    }
}
